zips/draft-pacu-regtest-local-consensus-definition.md

```diff
@@ -0,0 +1,313 @@
+```
+  ZIP: Unassigned
+  Title: Regtest: Definition of a Local Consensus test mode for Zcash Full-Nodes
+  Owners: ZIP Editors
+  Original-Authors: Francisco Gindre <pacu@zecdev.org>
+  Credits: Daira-Emma Hopwood
+           Zancas (Zingo Labs)
+           idky137 (Zingo Labs)
+           Kris Nuttycombe
+  Status: Draft
+  Category: Consensus
+  Created: 2025-02-26
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/986>
+```
+# Regtest: Definition of a Local Consensus test mode for Zcash Full-Nodes
+
+
+## Terminology 
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this document
+are to be interpreted as described in BCP 14 [^BCP14] when, and only when, they 
+appear in all capitals.
+
+The terms "Mainnet" and "Testnet" in this document are to be interpreted as defined 
+in the Zcash protocol specification [^protocol-networks].
+
+## Abstract
+Regtest is a testing mode inherited from the Bitcoin node. It provides a way to 
+have a consensus node on a local network with a private state that a developer 
+can control in order to reproduce certain situations deterministically for 
+testing purposes. Regtest, as it was implemented on Zcashd, is similar to a 
+“testnet” but without miners and remote peers.
```

Comment:
If I recall correctly many of the qa/rpc-tests in zcash acually have more than local "peer"..  I wonder whether the "remote-vs-local" distinction here, might be somewhat orthogonal to the purpose of regtest mode, which though frequently used with multiple nodes in the same ip/tcp space, could be used more "remotely" with the same regtest properties.